tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js

```diff
@@ -185,7 +185,9 @@ module.exports = {
             collaboratorResult[attrName] = false
           }
         }
-        await this.api.elementIdClick(collaboratorElementId)
+        this.moveToElement('@collaboratorAccessDetailsDrop', -9, 0)
+        this.api.mouseButtonClick()
+        this.waitForElementNotPresent('@collaboratorAccessDetailsDrop', 1000)
```

Comment:
don't we have some default timeout, so that we don't need to have the fixed `1000`here?

Reply:
added `waitForNegativeConditionTimeout`.